scripts/release/rpm/test_mariner_in_docker.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# This script should be run in a centos7 docker.
+set -exv
+
+export USERNAME=azureuser
+
+tdnf --nogpgcheck install /mnt/rpm/$RPM_NAME -y
+
+tdnf install git gcc python3-devel python3-pip findutils -y
```

Comment:
Can this script be reused from `test_rpm_in_docker.sh` by introducing some parameters?

Reply:
There are some difference between Mariner and UBI, see https://github.com/Azure/azure-cli/pull/25747#discussion_r1132030505.

Using same script decreases the duplication code but increases the complexity in script.
We've used different dockerfile for Mariner and UBI. We can also use different test scripts.